src/locale/de_DE.ts

```diff
@@ -252,7 +252,13 @@ export default {
       minScaleColumnTitle: 'Min. Maßstab',
       maxScaleColumnTitle: 'Max. Maßstab',
       amountColumnTitle: 'Anzahl',
-      duplicatesColumnTitle: 'Doppelte'
+      duplicatesColumnTitle: 'Doppelte',
+      ruleMoveUpTip: 'Verschiebe Regel eine Position nach oben',
+      ruleMoveDownTip: 'Verschiebe Regel eine Position nach unten'
+    },
+    GsReorderButtonGroup: {
+      ruleMoveUpTip: 'Move rule one position up',
+      ruleMoveDownTip: 'Move rule one position down'
```

Comment:
Minor: Please use respective language. See also es_ES.ts

Reply:
What do you exactly mean by that?

Reply:
Oh sorry, now I see the mistake :see_no_evil: . I'll come up with a correction

Reply:
See 75eb82d